src/common/datepicker-base.ts

```diff
@@ -62,11 +62,4 @@ export abstract class DatePickerBase implements ControlValueAccessor {
   }
 
   public abstract writeValue(value: any);
-
-  public isSameDate(date1: moment.Moment, date2: moment.Moment) {
```

Comment:
I think this function needs to stay in in order to maintain the fix for the min/max date issue